doc/dev/zoned-storage.rst

```diff
@@ -34,21 +34,21 @@ Currently we can perform basic RADOS benchmarks on an OSD running on an HM-SMR
 drives, restart the OSD, and read the written data, and write new data, as can
 be seen below.
 
-Please contact Abutalib Aghayev <agayev@cs.cmu.edu> for questions.
+Please contact Abutalib Aghayev <agayev@psu.edu> for questions.
 
 ::
    
-  $ zbc_info /dev/sdc
+  $ sudo zbd report -i -n /dev/sdc 
```

Comment:
nit, might want to drop the trailing space?

Reply:
Done.